### PR TITLE
MS: add eslint to dev dependencies

### PR DESCRIPTION
**Pull request by nailaghigo**
Added ESlint to dev dependencies.

### CODE REVIEW

**Review by fjmorabito — APPROVED** [review]
Great way to implement prettier. Don't know if its necessary more fields declaration, like adding some in rules, but it is awesome that you have implemented Eslint for the gruop project.

File: package.json [hunk 1]
```diff
@@ -19,10 +19,15 @@
...
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "nodemon": "^2.0.14"
+    "eslint": "^8.2.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-google": "^0.14.0",
+    "eslint-plugin-import": "^2.25.3",
+    "nodemon": "^2.0.14",
+    "lint": "eslint."
```
Comment (aquilesmariani) [inline package.json:31]: I think this was meant to be a script, try moving it up to the `scripts` section and run it to check it's working properly.

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
Just one small adjustment. Thanks for adding eslint!

File: package.json (hunk 1 above)
Comment (nailaghigo) [reply package.json]: I only added line 31, lines 28-29 and 30 were added automatically

**Review by aquilesmariani — APPROVED** [review]
Great work Nai! Looks good to me

**Pull request merged**